app/models/distribution.rb

```diff
@@ -33,7 +35,7 @@ class Distribution < ApplicationRecord
 
   validates :storage_location, :partner, :organization, presence: true
   validate :line_item_items_exist_in_inventory
-
+  validate :issued_at_end_is_after_issued_at
```

Comment:
```suggestion
  validate :issued_at_end_is_after_issued_at, if: 'issued_at_timeframe_enabled.present?'
```
This puts the conditional in the validate rather than in the method